test/main/call_test.cpp

```diff
@@ -0,0 +1,24 @@
+#include "main_test_helper/main_test_helper.h"
+
+using namespace kuzu::common;
+using namespace kuzu::testing;
+
+TEST_F(EmptyDBTest, SetThreadForExecution) {
```

Comment:
Open an issue for you to remove these 2 tests once we support `RETURN num_execution_thread()`. Then you can reuse the testing infrastructure.